settings.py

```diff
@@ -126,3 +133,12 @@
 #     "password ": "chaos",
 #     "db ": "db"
 # }
+
+SOURCE_PARTS_NAME = {
```

Comment:
Should be an `OrderedDict`. Otherwise, some generalizations could end up before the specifics (e.g. what is `misc` ends up first?).